sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure Key Vault Secret client library for Python
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings. Secret client library allows you to securely store and tightly control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge,backup, restore and and list the secrets and its versions.
+
+Use the secret client library to create and manage secrets.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets) | [Package (Pypi)](TODO) | [API reference documentation](TODO) | [Product documentation](TODO)
+## Getting started
+### Install the package
+Install the Azure Key Vault client library for Python with [pip](https://pypi.org/project/pip/):
+
+`$ pip install azure-security-keyvault
+`
+
+#### Prerequisites
+* An [Azure subscription](https://azure.microsoft.com/free/).
+* Python 2.7, 3.4 or later to use this package.
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with this Azure CLI command:
+
+    `az keyvault create --resource-group <resource-group-name> --name <key-vault-name>`
+
+### Authenticate the client
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyVaultClient](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault) class. You need a vault url, client secret credentials (client id, client secret, tenant id) and a [resource url](https://vault.azure.net) to instantiate a client object.
+
+ #### Create/Get credentials
+Use the [Azure CLI](https://docs.microsoft.com/cli/azure) snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+
+     `az ad sp create-for-rbac -n <your-application-name> --skip-assignment`
+    > If you don't specify a password, one will be created for you.
+* Grant the above mentioned application authorization to perform secret and key operations on the keyvault:
+     `az keyvault set-policy --name <key-vault-name> --spn <your-service-principal-id> --secret-permissions <secret-permissions> --key-permissions <key-permissions>`
+
+#### Create Secret client
+Once you have the `client_id`, `client_secret` and `tenant_id` from above, you can create the [SecretClient](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/_client.py):
```

Comment:
Are client_id, client_secret and tenant_id values people are supposed to replace in this code snippet? We are supposed to encourage people to write secure code and not embed credentials in code but as environment variables